litmus-portal/frontend/src/components/Sections/Settings/TeammingTab/styles.ts

```diff
@@ -0,0 +1,136 @@
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  UMDiv: {
+    marginTop: theme.spacing(3.75),
+    marginLeft: theme.spacing(7.5),
+  },
+  headerText: {
+    fontSize: '1.5625rem',
+  },
+  root: {
+    maxWidth: '63.75rem',
+  },
+  members: {
+    display: 'flex',
+    justifyContent: 'flex-start',
+    marginTop: theme.spacing(2.625),
+    marginBottom: theme.spacing(1.875),
+    color: theme.palette.secondary.dark,
+  },
+  descText: {
+    marginBottom: theme.spacing(3.75),
+    fontSize: '1rem',
+  },
+  memTypo: {
+    fontSize: '1rem',
+  },
+
+  table: {
+    maxWidth: '63.75rem',
+    maxHeight: '31.25rem',
+    border: '1px solid rgba(0,0,0,0.05)',
+  },
+  filterOpt: {
+    '&:hover': {
+      background: 'rgba(16, 155, 103, 0.1);',
+    },
+  },
+  toolbar: {
+    maxWidth: '63.75rem',
+    height: '6.125rem',
+    border: '1px solid rgba(0,0,0,0.05)',
+    marginBottom: theme.spacing(1.25),
+  },
+  filter: {
+    display: 'flex',
+    alignItems: 'center',
+    marginLeft: theme.spacing(5),
+  },
+  filterMenu: {
+    marginLeft: theme.spacing(1.25),
+    border: '1px solid rgba(0,0,0,0.05)',
```

Comment:
use the customColors from theme...it has a black color where you can pass the opacity